iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -155,6 +190,20 @@ static Value buildFlowWorkgroupCountOp(OpBuilder &b, unsigned dim) {
       b.getInsertionPoint()->getLoc(), dim);
 }
 
+// Explcitly list the constant we want to clone inside the dispatch region
+// rather than pass as arguments.
+// TODO(nicolasvasilache): add more types of constants that we wish to support.
```

Comment:
fyi: the RematerializeDispatchConstants pass does this kind of thing for the old dispatches but hasn't yet been updated to the new form - when it is we could just rely on that so it can run after dispatch region formation but before outlining (there's a lot of CSE/etc that can happen between the two).

Reply:
Cool, updated the TODO to drop this when ready.